.gitignore

```diff
@@ -0,0 +1,25 @@
+/Makefile
+/Makefile.in
+/aclocal.m4
+/autom4te.cache/
+/config/
+/config.log
+/config.status
+/configure
+/extensions/CoverflowAltTab@dmo60.de/Makefile
+/extensions/CoverflowAltTab@dmo60.de/Makefile.in
+/extensions/CoverflowAltTab@dmo60.de/schemas/org.gnome.shell.extensions.coverflowalttab.gschema.valid
```

Comment:
Consider adding just `*.gschema.valid` ignore rule instead, so you don't have to list all the `.gschema.valid` files here.